great-idea-website/css/style.css

```diff
@@ -0,0 +1,204 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+a   {
+    color: #222;
+    text-decoration: none;
+}
+
+header  {
+    position: relative;
+    height: 60px;
+}
+
+nav     {
+    display: inline-block;
+}
+
+#logo   {
+    display: inline-block;
+    position: absolute;
+    top: 0;
+    right: 200px;
+    padding-top: 15px;
+}
+
+ul   {
+    margin: 0;
+    padding: 0;
+    ;list-style-type: none;
+}
+
+ul li   {
+    display: inline-block;
+}
+
+ul li a     {
+    display: inline-block;
+    padding: 20px 15px;
+    margin-right: 3em;
+    color: grey;
+    font-weight: 500;
+
+}
+
+main    {
+    width: 85%;
+    margin: 0 auto;
+    padding: 20px;
+}
```

Comment:
You might want to add a max-width in px to make sure it looks good as well on bigger screen

Reply:
okay, will do this in the next project.